sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -63,8 +63,44 @@ With the value of the `endpoint`, `credential` and a `region`, you can create th
 <!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
 
 ```python
+from azure.ai.translation.text import TextTranslationClient, TranslatorCredential
+
+endpoint = os.environ["AZURE_TEXT_TRANSLATION_ENDPOINT"]
+apikey = os.environ["AZURE_TEXT_TRANSLATION_APIKEY"]
+region = os.environ["AZURE_TEXT_TRANSLATION_REGION"]
+
 credential = TranslatorCredential(apikey, region)
-text_translator = TextTranslationClient(credential, endpoint)
+text_translator = TextTranslationClient(endpoint=endpoint, credential=credential)
+```
+
+<!-- END SNIPPET -->
+
+#### Create a `TextTranslationClient` using Entra (Azure Active Directory) Authentication
+
+`AzureKeyCredential` authentication is used in most of the examples, but you can also authenticate with Entra (Azure Active Directory) using the [Azure Identity library][azure_identity].  Note that regional endpoints do not support AAD authentication.
+
+Create a [custom subdomain][custom_subdomain] for your resource in order to use this type of authentication.
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
+
+```powershell
+pip install azure-identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and [grant access][aad_grant_access] to your Translator resource by assigning the `"Cognitive Services User"` role to your service principal.
+
+Set the values of the `client ID`, `tenant ID`, and `client secret` of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
```

Comment:
Checking the code snippet below, the variables "`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`." are not set. Should those be set?

Reply:
They're expected to be in the execution environment as environment variables.  We could add an example showing how to do that in Powershell or setting them in the .env file.